ibis-server/tests/routers/v2/connector/test_clickhouse.py

```diff
@@ -529,6 +529,15 @@ def test_metadata_list_constraints(clickhouse: ClickHouseContainer):
         result = response.json()
         assert len(result) == 0
 
+    def test_metadata_db_version(clickhouse: ClickHouseContainer):
+        connection_info = _to_connection_info(clickhouse)
+        response = client.post(
+            url=f"{base_url}/metadata/version",
+            json={"connectionInfo": connection_info},
+        )
+        assert response.status_code == 200
+        assert response.text is not None
```

Comment:
Is it possible to assert the version text? or is it not a static value?

Reply:
We use the `head` tag. It is not a static version.